mvp.feature

```diff
@@ -896,11 +942,149 @@ Feature: MVP features for NVA
+    Given that the Creator Opens a DOI request entry from My Worklist
+    When they click the Edit publication button
+    Then the Wizard is opened on the first tab
+
+  @1240
+  Scenario: Creator deletes a DOI request
+    Given that the Creator opens My Worklist
+    When they click the Delete button on a DOI request
+    Then the request is deleted from their work list
+    And the request is deleted from the work list of their curator
+    And the Public Page for Publication has an enabled "Request DOI" button
+
+  @1242
+  Scenario: Owner navigates to the Public Page for Publication for a Publication with an existing DOI
+    Given that the Creator navigates to the Public Page for Publication
+    And they are the Owner of this Publication
+    And the Publication has a DOI
+    When they see the Status Bar
+    Then they see that the Request/Draft DOI button is disabled
+  # a Publication may have more then one DOI,
+  # but NVA is only source for one DOI for each Publication
+
+  @1231
+  Scenario: Owner navigates to the Public Page for Publication for published publication without DOI
+    Given that the Creator navigates to the Public Page for Publication
+    And they are the Owner of this Publication
+    And the Publication has no DOI
+    When they see the Status Bar
+    Then they see buttons for Request a DOI and Edit Publication
+
+  @511
+  Scenario: Owner navigates to the Public Page for Publication of a Publication and opens the "Request a DOI" dialog
+    Given that a Creator navigates to the Public Page for Publication for published publication without DOI
+    And they are the Owner of this Publication
+    When they click the "Request a DOI" button
+    Then the "Request a DOI dialog" is opened
+    And they see fields for Message
+    And a button for Send Request
+
+  @1232
+  Scenario: Owner navigates to the Public Page for Publication and requests a DOI
+    Given that the Creator navigates to the Public Page for Publication
+    And they are the owner of the publication
+    And open "Request a DOI" dialog
+    And optional add a message to the curator
+    When the user click the send button
+    Then the Public Page for Publication is displayed
+    And the "Request a DOI" button is renamed to "DOI pending" and is disabled
+    And the request is listed in user work list
+    And the request is listed in curator work list
+
+  @1233
+  Scenario: Owner navigates to the Public Page for Publication for unpublished publication without DOI
+    Given that the Owner view Public Page for Publication for publication
+    And the publication is not Published
+    And the publication has no DOI
+    When they look at the Status Bar
+    Then they see buttons for Draft a DOI and Edit publication
+
+  @1234
+  Scenario: Owner navigates to the Public Page for Publication and selects Draft a DOI
+    Given that the Owner View Public Page for Publication for unpublished publication without DOI
```

Comment:
Rephrase to match the phrases used in previous scenarios.  In general, we use many variations of the same two phrases which makes it difficult to understand the setting. 

Case 1: A Creator views the Public Page of an owned Published Publication
Case 2: A Creator views the Public Page of an owned Draft Publication

If we setup the background then we could also refer to an Owner who is a Creator that owns the scenario's publication and if necessary, the NotOwner who is a Creator that does not own the scenario's publication